special-interest-groups/infrastructure/meetings/2023-01-x.md

```diff
@@ -0,0 +1,47 @@
+# Infrastructure SIG Meeting 2023-01-x
+
+Post meeting -> [Link to recorded meeting]()
+
+Meeting shall be ~30 min. Items not covered will be left for subsequent meetings. 
+
+## Agenda 
+
+#### Inclusiveness Statement
+
+- All members are welcome to share ideas / suggestions / feedback 
+- All ideas / suggestions / feedback will be treated with respect 
+- Failure to meet above will result in not attending future meetings
+
+#### SIG Charter 
+
+- See [charter](../charter.md)
+- Meeting format 
+  - Frequency -> 1 per quarter / 1 month
+  - Try to keep to 30 min 
+- Tracker frontend as a separate SIG?
```

Comment:
Tracker frontend should either be specific to SIG Infrastructure or should be a Working Group (temporary cross-group collab) between SIG Infrastructure and a TBD group focused on design or UX. Likely it should stay with SIG Infrastructure and some other group can lay out specs and guidelines on how to develop UI/UX

Reply:
Yeah that is something that I wanted to raise with participants. But I agree that UI / UX should be under another group.